core/Cargo.toml

```diff
@@ -17,6 +17,7 @@ thiserror = "1"
 constants = { path = "../constants" }
 regex = "1.6.0"
 serde = { version = "1", features = ["derive"] }
+serde_stacker = "0.1"
```

Comment:
Oh I just meant for the places I pointed out, we can't use it. However, it was used when deserializing into the `Translation` and `Namespace` types.

Reply:
I was wondering about that because now we have that test that checks for unused dependencies that would have failed if you weren't using `serde_stacker`.